web/src/tippyjs.ts

```diff
@@ -502,10 +503,20 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#user_info_popover .status-emoji",
+        target: ".user-card-status-text .status-emoji",
         appendTo: () => document.body,
     });
 
+    delegate("body", {
+        target: ".status-emoji-name",
+        placement: "bottom",
+        delay: INSTANT_HOVER_DELAY,
+        appendTo: () => document.body,
+        onShow() {
+            hide_all();
```

Comment:
So I think this is not the right approach; [Tippy has its own hideAll() method](https://atomiks.github.io/tippyjs/v6/tippy-instance/) we could use. I think the better approach here would be to find [the Tippy instance](https://atomiks.github.io/tippyjs/v6/tippy-instance/) shown on the username, and then [hide](https://atomiks.github.io/tippyjs/v6/methods/#hide) it while the status emoji tooltip is being shown. Once the status emoji tooltip is hidden, then again [show](https://atomiks.github.io/tippyjs/v6/methods/#show) the username Tippy.

Reply:
Thanks @karlstolley , I thought about the above approach as well but the main issue with the above was to locate the instance associated with the user-name in the .status-emoji-name delegate function so that I can hide it. This is because the tooltip is shown for the complete ```user-presence-link``` which includes user-name as well as status-emoji.
Was a little busy with college stuffs so couldn't look up. Will try to fix this ASAP.